functions/autorun.lua

```diff
@@ -54,14 +54,15 @@ autoRunCodeEventFrame.OnEventFunc = function(self, event)
     if (event == "PLAYER_SPECIALIZATION_CHANGED") then
         --create a trigger for the event, many times it is triggered more than once
         --so if the event is triggered a second time, it will be ignored
-        local newTimer = C_Timer.NewTimer(1, function()
-            Details:DispatchAutoRunCode("on_specchanged")
-            
+		local spec_changed_func = function()
```

Comment:
These changes do little to change the functionality of these functions. They reduce readability imo.